### PR TITLE
CCP Red Dragon/Gate + Turret docs

### PR DESCRIPTION
**Pull request by CCP-Red-Dragon**
Documentation of steps for the Smart Gate and Smart Turret examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Welcome, this repository contains guides and examples to get started building on
 To start building, follow the steps below to setup your local development tools and environment. If you already have the tools, make sure they are the correct version as otherwise you may have difficulties running the examples and building.
 
 ### Step 0: Installing general tools
-Before you get started you need to either install, or make sure you have the required tools.
+Before you get started you need to either install, or make sure you have the required tools. This guide is for Linux, if you use a different OS then visit https://docs.evefrontier.com/Tools and follow the guide for your operating system.
```
Comment (0xxlegolas) [inline README.md]: This guide is for Linux,
This gives a misinterpretation that this guide is only of Linux, I guess we should say install these tools for linux and other os they can refer to a different section

File: README.md [hunk 2]
```diff
@@ -80,21 +80,25 @@ docker compose cp world-deployer:/monorepo/abis .
 
 ### Step 0.5: Start Building!
 
-Now that your local tools and development environment is set up, you're ready to start building! To begin, simply navigate to the desired example directory (or use the links below) then follow the instructions outlined in its README file.
+Now that your local tools and development environment is set up, you're ready to start building! To begin, simply navigate to the desired example directory (or use the links below) then follow the instructions outlined in its README file. For more information on Smart Assemblies you can visit [this page](https://docs.evefrontier.com/SmartAssemblies) on the documentation.
```
Comment (0xxlegolas) [inline README.md:83]: love this

**Review by 0xxlegolas — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (CCP-Red-Dragon) [reply README.md]: Good point, have fixed.

File: README.md (hunk 2 above)
Comment (CCP-Red-Dragon) [reply README.md:83]: Thanks :)

**Review by 0xxlegolas — APPROVED** [review]

**Pull request merged**